dla_cnn/desi/DesiMock.py

```diff
@@ -88,6 +95,18 @@ def get_data(start_point = 0, end_point = self.data_size):
             sightline.dec = self.data[id]['DEC']
             sightline.dlas = self.data[id]['DLAS']
             sightline.loglam = np.log10(self.wavelength[start_point:end_point])
+
+            wavelength = self.wavelength[start_point:end_point]
+            dlambda = wavelength-np.roll(wavelength,1)
+            test = (dlambda>0)[1:]
+            indice = np.argwhere(~test)
+            
+            if indice.size:# the wavelength array may not be monotonic increasing, so we do a test here.
```

Comment:
Why would it not be monotonically increasing?  
If it isn't this should mean there is a problem and 
you should `raise` an error.

Reply:
I am not sure, only a very small part of the data have this problem. Maybe we can still use  these spectrum although they have this problem. I am not sure...

Reply:
Let's try to understand what this occurs.
Can you provide an example on Slack?

Reply:
Sure~

Reply:
This was a bug.